eskip/eskip.go

```diff
@@ -107,6 +245,15 @@ type Predicate struct {
 	Args []interface{} `json:"args"`
 }
 
+func (p *Predicate) String() string {
```

Comment:
this package has a predicateString function, we could use it inside this method

Reply:
fixed as discussed